apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -31,4 +31,9 @@ export class EventResolver {
   async deleteEvent(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
     return this.eventService.delete(id);
   }
+
+  @Mutation(() => Event)
+  async addParticipantToEvent(@Args('eventId') eventId: number, @Args('userId') userId: number) {
```

Comment:
Would it be possible to see if the user is already part of the event here? In case there's some edge case we miss. You can reach out to me if you're not sure how to implement it.

Reply:
The purpose of this is to add users to events not check if they're in them. If we try to add a user twice it will just add them once and if the user is in the event already it won't do anything. 

Given the above info, is there a benefit to checking if a user is in the event? We could always get the participants list from the event to check if a user is in it or not

Reply:
The only benefit I see is blocking the interaction with the database to add the user and maybe having a message to display on the frontend. But I don't think it's necessary and you can keep it as is. 👍